### DISCUSSION TITLE
I am using OpenCL to utilize the GPU, but I have not seen a performance improvement after running it.

### DISCUSSION THREAD

**Discussion by lqy1992** [ImageMagick/ImageMagick#7508] [Development]
### ImageMagick version

7.1.1-35 Q16-HDRI x86_64

### Operating system, version and so on

centos7.9

### Description

I am using OpenCL to utilize the GPU, and I can confirm that the processing is running on the GPU. However, through stress testing, I have noticed that while the CPU usage has not decreased, the GPU usage has significantly increased. Despite this, the performance of image processing has not improved. Could this be due to an issue with my installation?

[root@3d-imagick-php-178-2-168-192-125 application]# convert --version
WARNING: The convert command is deprecated in IMv7, use "magick" instead of "convert" or "magick convert"

Version: ImageMagick 7.1.1-35 Q16-HDRI x86_64 1bfce2a62:20240713 https://imagemagick.org
Copyright: (C) 1999 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC HDRI OpenCL OpenMP(4.5) 
Delegates (built-in): bzlib fontconfig freetype jng jpeg lcms ltdl lzma openexr pangocairo png raqm tiff x xml zlib zstd
Compiler: gcc (8.3)

![image](https://github.com/user-attachments/assets/24427b68-8d17-411f-af1b-9a07d5017107)
![image](https://github.com/user-attachments/assets/24055306-2dc1-4897-a040-56f1d9a3ad10)

**Comment by urban-warrior** [comment]
Accelerated resizing is supported only for the following filters: Box, Triangle, Hann, Hamming, Blackman, Cubic, SincWeight, and SincFast. The default filters are Lanczos or Mitchell. Try using one of the supported filters to experience the expected performance improvement.

**Comment by superuser-miguel** [comment]
Is Accelerated coversion possible ? I am trying to conver jpg to avif and hoping to get better results with a GPU. At the moment I am using gnu parallel to get all cores going.

**Reply by fmw42** [reply]
I do not believe that conversion between formats is accelerated by OpenCL in Imagemagick.
